codelabs/gke-kayenta-workshop/services/backend/main.go

```diff
@@ -0,0 +1,137 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"os"
+	"strconv"
+	"time"
+
+	gce "cloud.google.com/go/compute/metadata"
+
+	"golang.org/x/oauth2/google"
+
+	"google.golang.org/api/monitoring/v3"
+)
+
+var count = 1
+var lastTime = time.Now()
+
+// Started with: https://github.com/GoogleCloudPlatform/golang-samples/blob/master/monitoring/custommetric/custommetric.go
+func writeTimeSeriesValue(now string, s *monitoring.Service, metricType string, metricValue int64) error {
+	project_id, err := gce.ProjectID()
```

Comment:
Worth checking these errors?

Reply:
Probably. There are a bunch of things I'd like to change in this driver. I dropped a TODO.